### ISSUE TITLE
Supports going 2x speed under part

### ISSUE THREAD

**Issue by ssjbardock** [supermerill/SuperSlicer#3704]
### What happened?

Underneath the floating part, supports are going 120mm/s, despite being set to 60mm/s (technically 50mm/s default speed with 120% set for support speed)

Can't tell why and can't find how to stop this.
![why faster 3](https://user-images.githubusercontent.com/18308623/223789758-676f05bb-8608-41d3-abe9-43342f54c18c.jpg)
![why fast 2](https://user-images.githubusercontent.com/18308623/223789759-770cd4bf-c371-426d-a0cc-730b0e4384d2.jpg)
![why fast](https://user-images.githubusercontent.com/18308623/223789761-027efe96-14e6-455d-8ab9-cdfa55646e37.jpg)

### Project file & How to reproduce

Float a part or use raft
Supports under the part are going double speed of what is listed.

### Version

2.5.59.0

### Operating system

Windows 10

**Comment by neophyl** [comment]
Please attach a zipped up project file that has this issue as requested when opening a new issue.  

For example if I use my normal profiles and add a box to a raft then I'm getting the correct speed for the raft as I have it configured.  This is why attaching a project file  is so important, its a snapshot of ALL your settings and makes it potentially possible to track down a cause.  Without one it is very difficult.

**Comment by ssjbardock** [comment]
> Please attach a zipped up project file that has this issue as requested when opening a new issue.
> 
> For example if I use my normal profiles and add a box to a raft then I'm getting the correct speed for the raft as I have it configured. This is why attaching a project file is so important, its a snapshot of ALL your settings and makes it potentially possible to track down a cause. Without one it is very difficult.

Updated original post with the project file 'too fast supports.zip'.

**Comment by neophyl** [comment]
You are fast :)  uploaded while I was editing. see my last comment.

**Comment by ssjbardock** [comment]
QUOTE: Edit - scratch that, I think its just treating the percentage value as a speed value. Change it to a different percentage and the speed appears to change to the same value in mm/s

I thought that may be the case, but then there's plenty of supports withing the object area all going 60mm/s.

**Comment by neophyl** [comment]
Yeah but all of those are when the actual model is being printed.  The speed change only occurs ones it gets to the object.  
Btw you have some parts of the object with zero support that are getting printed on air.
![image](https://user-images.githubusercontent.com/46955527/223792751-8ca93ea4-199b-4ae1-8f22-299949323150.png)

**Comment by neophyl** [comment]
![image](https://user-images.githubusercontent.com/46955527/223794545-f941dc94-a2f6-47ce-b4be-78c3577b1d8d.png)
Adding a box to the plater (not even part of the object) and the support speed starts to behave again.

**Comment by ssjbardock** [comment]
That did not work for me unfortunately.


![why fast 7](https://user-images.githubusercontent.com/18308623/223795065-8ac3a721-f4b2-4ebe-8e06-ae82336c232d.jpg)

**Comment by ssjbardock** [comment]
So it's not all empty layers, as I left some space between the cube top and part bottom, but it still kept the lower speed.

![image](https://user-images.githubusercontent.com/18308623/223796008-b00b63cf-c191-49b4-9195-eeda642ec150.png)

**Comment by ssjbardock** [comment]
Well at least I can print with a workaround. Unless anyone else finds a magical 'empty layers sometimes ignore your setting' checkbox, pretty safely assume it's a bug.

Thanks for help testing!
